anasnadeemws/react-template
# fix: optipng module issue on M1 & M2

### Ticket Link

---

### Related Links

---

### Description
optipng-bin module not found error on M1 Pro
---

### Steps to Reproduce / Test
1. Clone the repo in M1 Pro
2. yarn install (Even tho everything will get successful you'll see some warning error from optipng )
3. yarn start (Error from optipng-bin or pngbinquant will occur. Refer to ss below)
4. Try fixing the error by adding that module
5. You'll still get the error from import of png file logo in Header component (Refer ss below. Also try commenting it out to verify this)
<img width="856" alt="Screenshot 2024-03-06 at 11 20 12 AM" src="https://github.com/anasnadeemws/react-template/assets/161311819/94f56995-e669-4322-9144-0a69f4bdee48">
<img width="864" alt="Screenshot 2024-03-06 at 11 21 29 AM" src="https://github.com/anasnadeemws/react-template/assets/161311819/1311a73c-1288-4657-bd43-97f5ebc5dea7">

---

---

### Checklist


### GIF's

---

### Live Link

---

- http://wednesday-react-template.s3-website.ap-south-1.amazonaws.com/<BRANCH_NAME>
